preplan-backend/controllers/eventController.js

```diff
@@ -373,9 +372,149 @@ router.get('/position/:id', [authJwt.verifyToken, authJwt.isOrganizer], (req, re
         });
 })
 
-router.post('/create/timeSlot', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
+router.get('/edition/:id/registered', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
+    const editionId = req.params.id;
+    Volunteer.findOne({ edition: editionId })
+        .populate([
+            {
+                path: 'profile',
+                model: 'Profile',
+                populate: [
+                    {
+                        path: 'user',
+                        model: 'User'
+                    },
+                    {
+                        path: 'emergencyContact',
+                        model: 'EmergencyContact'
+                    }
+                ]
+            },
+            {
+                path: 'edition',
+                model: 'Edition',
+                populate: {
+                    path: 'event',
+                    model: 'Event'
+                }
+            },
+            {
+                path: 'availabilities',
+                model: 'Availability'
+            },
+        ])
+        .exec((err, vol) => {
+            if (err) {
+                res.status(500).send({ message: err });
+                return
+            }
+            const picked = (({ inscriptionDate, status, plannerId }) => ({ inscriptionDate, status, plannerId }))(vol);
+            picked['profile'] = sanitizeProfile(vol.profile);
+            picked['edition'] = sanitizeEdition(vol.edition);
+            picked['availabilities']
+            picked['id'] = vol['_id'];
+            res.status(200).json(picked);
+        })
+});
+
+router.post('/event/register', [authJwt.verifyToken, authJwt.isOrganizer], async (req, res) => {
```

Comment:
would like to keep the term inscription please, as registering is creating an account on the website

Reply:
Alrighty